client/app/components/ParameterValueInput.less

```diff
@@ -5,9 +5,12 @@
 .parameter-input {
   display: inline-block;
   position: relative;
+  width: 100%;
 
-  .@{ant-prefix}-input[type="text"] {
-    width: 195px;
+  .@{ant-prefix}-input,
+  .@{ant-prefix}-input-number,
+  .@{ant-prefix}-select {
```

Comment:
Suggestion for future, groups these selectors into a less variable `@ant-inputs` (if that's an actual less feature).

Reply:
Note: Just tried that, doesn't seem to work ^^. Also, in this case I don't want Date Picker parameters to be included (the transformation effect thing).